examples/with-emotion/package.json

```diff
@@ -12,13 +12,13 @@
   "author": "Thomas Greco",
   "license": "MIT",
   "devDependencies": {
-    "@emotion/babel-preset-css-prop": "10.0.27"
+    "@emotion/babel-preset-css-prop": "11.0.0"
```

Comment:
@emotion/babel-plugin if you drop css-prop

Reply:
Thanks, good catch! Updated in [`3e90856`](https://github.com/vercel/next.js/pull/18620/commits/3e90856e1703734ee32b60476e6f805a8870e8d2)